components/brave_wallet_ui/components/extension/confirm_send_transaction/confirm_send_transaction.tsx

```diff
@@ -143,6 +150,10 @@ export function ConfirmSendTransaction() {
     gasFee,
   ])
 
+  const isBraveWalletOrigin = React.useMemo(() => {
```

Comment:
nit:

It would be nice to refactor this logic out from the confirm_send_transaction panel and reuse it across other panels. I don't believe we initiate other types of transactions within wallet at this point, but something to consider if we ever do start doing that.